nixos/doc/manual/development/option-types.section.md

```diff
@@ -211,6 +211,14 @@ merging is handled.
     definitions cannot be merged. The regular expression is processed
     using `builtins.match`.
 
+## Network types {#sec-option-types-network}
+
+Types used in the network library.
+
+`types.ipv6AddrStr`
```

Comment:
```suggestion
`types.ipv6Addr`
```
I think we should just use the name without str since there is no other version we need to differentiate from.